quantum/keymap_extras/keymap_slovenian.h

```diff
@@ -161,11 +161,3 @@
 #define SI_LCBR ALGR(SI_B)    // {
 #define SI_RCBR ALGR(SI_N)    // }
 #define SI_SECT ALGR(SI_M)    // §
-
-// DEPRECATED
-#define SI_QOT  SI_QUOT
-#define SI_SV   SI_SCAR
-#define SI_CV   SI_CCAR
-#define SI_ZV   SI_ZCAR
-#define SI_DQOT SI_DQUO
-#define SI_QST  SI_QUES
```

Comment:
If you want to remove these, then this PR will need to target `develop` and go through breaking changes. You'll also have to check for any other usages of these defines in other keymaps, and get their signoff too.

Reply:
I've looked for other usages and there aren't any, only in my own keymaps.

Reply:
It still has to go through `develop` though, since it's possible some people are using these defines out-of-tree.

Reply:
I have removed this to prevent any unwanted changes.